test/framework/ippool.go

```diff
@@ -0,0 +1,81 @@
+package framework
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+type E2EIPPool []string
```

Comment:
nit: I would vote to move `E2EIPPool` and its receiver methods to `networkutils` package

Reply:
Fine by me :)